### PR TITLE
MLCOMPUTE-707 | fix unreadable formatting in costs log

### PR DESCRIPTION
**Pull request by CaptainSame**
Fix unreadable cost log in tron logs

Testing:
With DRA enabled:
image

With DRA disabled:
image

### CODE REVIEW

**Review by edingroot — APPROVED** [review]

File: service_configuration_lib/spark_config.py [hunk 1]
```diff
@@ -1073,15 +1073,19 @@ def compute_approx_hourly_cost_dollars(spark_conf, paasta_cluster, paasta_pool):
     if min_dollars != max_dollars:
         log.warning(
             TextColors.magenta(
-                f'\nThe requested resources are expected to cost a maximum of $ {TextColors.bold(str(max_dollars))} '
-                f'every hour and $ {TextColors.bold(str(max_dollars * 24))} in a day.\n',
+                TextColors.bold(
+                    f'\nThe requested resources are expected to cost a maximum of $ {str(max_dollars)} '
+                    f'every hour and $ {str(max_dollars * 24)} in a day.\n',
```
Comment (88manpreet) [inline service_configuration_lib/spark_config.py:1078]:
```suggestion
                    f'\nExpected maximum cost based on requested resources: ${str(max_dollars)}/hour or'
                    f'${str(max_dollars * 24)}/day.\nPlease monitor y/spark-metrics for memory and cpu usage to tune requested executor count config spark.executor.instances. \nFollow http://y/write-spark-job for optimization tips',
```

File: service_configuration_lib/spark_config.py [hunk 2]
```diff
@@ -1073,15 +1073,19 @@ def compute_approx_hourly_cost_dollars(spark_conf, paasta_cluster, paasta_pool):
     if min_dollars != max_dollars:
```
Comment (88manpreet) [inline service_configuration_lib/spark_config.py:1073]: max_str = " maximum" if min_dollars != max_dollars else ""

File: service_configuration_lib/spark_config.py [hunk 3]
```diff
@@ -1073,15 +1073,19 @@ def compute_approx_hourly_cost_dollars(spark_conf, paasta_cluster, paasta_pool):
...
             TextColors.magenta(
-                f'\nThe requested resources are expected to cost a maximum of $ {TextColors.bold(str(max_dollars))} '
-                f'every hour and $ {TextColors.bold(str(max_dollars * 24))} in a day.\n',
+                TextColors.bold(
+                    f'\nThe requested resources are expected to cost a maximum of $ {str(max_dollars)} '
+                    f'every hour and $ {str(max_dollars * 24)} in a day.\n',
+                ),
             ),
         )
     else:
```
Comment (88manpreet) [inline service_configuration_lib/spark_config.py:1082]: I think you can simplify message by removing this else condition.

**Comment by 88manpreet** [comment]
Would it be possible to add a test example snapshot?

**Review by 88manpreet — APPROVED** [review]

**Pull request merged**